src/ex4/server/helper/logger.js

```diff
@@ -0,0 +1,11 @@
+import { writeToFile, readFile } from './fileHandler.js'
+
+export const logger = async (req, res, next) => {
+    const currentDate = new Date().toJSON().slice(0, 10).replace(/-/g, '/');
+    let logs = await readFile("./server/data/log.json")
```

Comment:
reading the file to log is redundant and also time consuming (depending on OS availability and file size..)
you don't need to read the file if the only thing you do is to add an extra log.

Also, if you already did it, why not putting it after the condition as well? so if the condition is false you will save your time and resources and won't read the file the first place.

you can just use `fs.appendFile` to append more data to a file.